savoirfairelinux/sous-chef
# Fixed #703: improve version control of .po file

## Fixes #703 by lingxiaoyang

### Changes proposed in this pull request:

1. Drop English .po files.
2. Remove line numbers from the location strings. This is achieved by extending `django-admin makemessages` command, before an official solution is provided.
3. Disable the line wrap of translatable strings to improve file readability.
4. Replace {% blocktrans %} with {% blocktrans trimmed %} to avoid newline confusion.
5. Provide an example of translator comment.
6. Update developer instructions in i18n doc.
7. Update CONTRIBUTING.md

### Status


### How to verify this change

1. See if the doc update makes sense (or your comments on .po file versionning).
2. Test `python3 src/manage.py makemessages -l fr --no-wrap --add-location=file`. See if it works fine.
3. Check updated `django.po` files and see if there are any errors. Don't bother with `fuzzy` ones as they will be double-checked on Transifex.

### Deployment notes and migration

none

### New translatable strings


### Additional notes

none
